src/status_im/contexts/wallet/signals.cljs

```diff
@@ -0,0 +1,10 @@
+(ns status-im.contexts.wallet.signals
+  (:require [utils.re-frame :as rf]))
+
+(rf/reg-event-fx
+ :wallet/pending-transaction-status-changed
+ (fn [{:keys [db]} [{:keys [message]}]]
+   (let [details (js->clj (js/JSON.parse message) :keywordize-keys true)
+         tx-hash (:hash details)]
+     {:db (-> db (update-in [:wallet :transactions tx-hash] assoc :status :confirmed :blocks 1))})))
```

Comment:
We can remove the `->` and use `db` directly instead, since it's a thread-first of only one step.

BTW, why do we set blocks as 1? is it possible to set more than 1?

Reply:
yeah we will be updating the block number going forward. This is basically saying the transaction has been accepted and added to the first block. From there we do some calculations to get the next block. However that is for a follow up pr when we handle the animations etc for the various blocks on different networks.
I'll link you to the conversation in the chat and it will explain it.

Reply:
Thanks!!